src/hooks/usePanResponder.ts

```diff
@@ -65,12 +65,28 @@ const usePanResponder = ({
   const meaningfulShift = MIN_DIMENSION * 0.01;
   const scaleValue = new Animated.Value(initialScale);
   const translateValue = new Animated.ValueXY(initialTranslate);
-
   const imageDimensions = getImageDimensionsByTranslate(
     initialTranslate,
     SCREEN
   );
 
+  useEffect(() => {
```

Comment:
@LimitedDani Sorry that it takes so long to merge this one, I am unfortunately pretty busy and have very limited time for OS. But I need to ask couple of questions here:

1. Why do we need `Animated.parallel` having single animation in array?
2. Can you shortly describe the idea, because the approach looks a bit hacky for me. From what I understand we run this effect on every render (no `deps`) and where we say that zoom is over (`onZoom(false)`) and animate zoom back to initial scale. 
What was initial problem we are solving? Was it that image stacked after pinch gesture and we were about to return it to initial scale?